### ISSUE TITLE
Automatically remove test samples in training dataloader

### ISSUE THREAD

**Issue by whaowhao** [LaihoE/did-it-spill#5]
Is there plan on automatically remove test samples in training data-loader? That'd be super helpful, since that'd be mostly likely every user's next step for this package. Why not make it default :)

**Comment by LaihoE** [comment]
Yes it's on the roadmap. Not too sure about the most elegant implementation. Removing the samples from the test-loader should be fairly easy by overwriting the sampler. The problem with removing samples from the train-loader is that they are often more complicated, custom samplers etc, and overwriting them might become tricky.

Do you think it matters if the samples are removed from the test-loader instead?

**Comment by LaihoE** [comment]
Nvm. I think it should be just as easy to do it for both loaders.
